core/src/test/java/com/scalar/db/storage/dynamo/ItemSorterTest.java

```diff
@@ -264,10 +266,27 @@ public void sort_ItemsGivenWithDescOrderingText_ShouldSortProperly() {
   @Test
   public void sort_ItemsGivenWithDescOrderingBlob_ShouldSortProperly() {
     // Arrange
+    Random random = new Random(777);
```

Comment:
I'm just wondering if there is any benefit of using Random with a fixed seed here.
I feel it's better to use fixed hard-coded byte arrays so that we (people) can see what is the expected result.
What do you think?

Reply:
I see. I had been thought about using a loop for the test with many values, so I used Random. Actually, I see some libraries still using Random for testing, so I thought it's common. For example from [guava](https://github.com/google/guava/blob/8075df7ffd63b4b96cd0bdfdc2dde71d08f672c9/guava-tests/test/com/google/common/primitives/UnsignedBytesTest.java#L288-L306)
I made to use fixed hard-coded byte arrays in this [commit](https://github.com/scalar-labs/scalardb/pull/327/commits/0b85f8712e036c51b4f94a71e169b296892f494f). Thank you for your suggestion.

Reply:
It makes more sense if we test many cases using loop and random but the test was only comparing two byte-arrays, that's why I asked.

Reply:
That's right. So should we random and for loop here or keep it like the new commit I made?

Reply:
I think the current code is fine and consistent with other tests.
If you feel it's not tested well, we can add more tests with loop and random. (please add instead of update in such a case)

Reply:
@thongdk8 I will merge this. If you want add more tests with loop and random, please create a new PR for that. Thanks.

Reply:
Got it. Thank you.